python/nav/ipdevpoll/plugins/prefix.py

```diff
@@ -50,7 +50,10 @@
 from nav.ipdevpoll import Plugin
 from nav.ipdevpoll import shadows
 
-VLAN_PATTERN = re.compile(r"(Vl(an)?|irb\.|reth)(?P<vlan>\d+)", re.IGNORECASE)
+VLAN_PATTERN = re.compile(
+    r"(Vl(an)?|irb\.|reth(\d+)\.)(?P<vlan>\d+)",
```

Comment:
Good point from @hmpf: Capturing the digits immediately following the `reth` string is useless and just complicates the pattern unnecessarily.

```suggestion
    r"(Vl(an)?|irb\.|reth\d+\.)(?P<vlan>\d+)",
```